internal/reorder/pick.go

```diff
@@ -10,12 +15,37 @@ type PickCmd struct {
 	Commit string
 }
 
-func (b PickCmd) Execute(ctx *Context) error {
-	panic("not implemented")
+func (p PickCmd) Execute(ctx *Context) error {
+	err := ctx.Repo.CherryPick(git.CherryPick{
+		Commits: []string{p.Commit},
+		// Use FastForward to avoid always amending commits.
+		FastForward: true,
+	})
+	if conflict, ok := errutils.As[git.ErrCherryPickConflict](err); ok {
```

Comment:
Not sure if this is super bad practice. The alternative is doing
```go
var conflict git.ErrCherryPickConflict
if errors.As(err, &conflict) {
```
which is only one extra line.

Let me know if this burns your eyes to look at and I'll change it 😅.

Reply:
Really? I feel like I do this assign-in-if frequently to limit the scope. (Also mentioned in https://go.dev/doc/effective_go#if)

Reply:
Yeah, the only difference here is that we have to declare the concrete type of the error which isn't possible without using generics here. There's no
```go
if var conflict git.ErrCherryPickConflict; errors.As(err, &conflict) { ... }
```
construct in Go.

I guess I could also do
```go
if conflict := git.ErrCherryPickConflict{}; errors.As(err, &conflict) { ... }
```
but I've never seen that usage before.